src/node.h

```diff
@@ -375,8 +375,6 @@ NODE_DEPRECATED("Use WinapiErrnoException(isolate, ...)",
 })
 #endif
 
-const char *signo_string(int errorno);
```

Comment:
I was under the impression that ABI breakage would only occur when you change structs an the sorts, so people can't serialize/ deserialize. Sure it would break API tho. Academic in any case, since I'll just move the definition and leave the prototype there